charts/riff/values.yaml

```diff
@@ -1,21 +1,28 @@
 ---
 cert-manager:
   enabled: true
-keda:
-  enabled: true
-knative:
-  enabled: true
+## Uncomment the following to disable the chart even if the tag is specified
+# keda:
+#   enabled: false
+## Uncomment the following to disable the chart even if the tag is specified
+# knative:
+#   enabled: false
 kpack:
   enabled: true
 riff:
   build:
     enabled: true
   builders:
     enabled: true
-  runtimes:
-    core:
-      enabled: false
-    knative:
-      enabled: false
-    streaming:
-      enabled: false
+## Uncomment any runtime to install the chart even if the corresponding tag isn't specified
+#  runtimes:
+#    core:
+#      enabled: true
+#    knative:
+#      enabled: true
+#    streaming:
+#      enabled: true
+tags:
```

Comment:
let's also move all of the build related components under a `build` tag that is enabled by default

Reply:
ok, sounds good